doc/source/templates/04_finetuning_llms_with_deepspeed/finetune_hf_llm.py

```diff
@@ -34,10 +35,42 @@
 OPTIM_BETAS = (0.9, 0.999)
 OPTIM_EPS = 1e-8
 OPTIM_WEIGHT_DECAY = 0.0
+TARGET_LLM_TO_NUM_SELF_ATTENTION_LAYERS = {
```

Comment:
These numbers are un-intuitive? We need to automate this maybe based on what the base attention model is called?